developer/swag-shop/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Building & Launching a Swag Shop on Reaction
+
+---
+
+ **⚠ NOTICE:** This tutorial aggregates a couple of Reaction blog posts into one place that should serve our community as an accessible show case on how to build a fully customized shop on top of Reaction. The orginal blog posts can be found here:
+- <https://blog.reactioncommerce.com/building-and-launching-a-store-on-reaction/>
```

Comment:
Change this part to:
- [Building & Launching a Swag Shop on Reaction](https://blog.reactioncommerce.com/building-and-launching-a-store-on-reaction)
- [Landing Page](https://blog.reactioncommerce.com/building-launching-a-swag-shop-part-2/)
- [Landing Page, Part 2](https://blog.reactioncommerce.com/building-a-swag-shop-lp2/)
- [Category Tiles & Related Products](https://blog.reactioncommerce.com/building-a-swag-shop-category-tiles/)

Reply:
The notice and the links are referring to the blog posts. Should we remove both together (the NOTICE and the link list?)